docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -115,4 +111,3 @@ This limit will be applied to every container of the given component.
 +
 For the `pugin` component type, CPU requests can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-If none of them are specified, system-wide defaults are applied (see description of `pass:[CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__REQUEST__CORES]` system property).
```

Comment:
Same as the other case, we should remove this and add that "if they are not specified, the values are undetermined: they may (or may not) be inferred from the application that consumes the devfile or from Kubernetes." (reword if needed)